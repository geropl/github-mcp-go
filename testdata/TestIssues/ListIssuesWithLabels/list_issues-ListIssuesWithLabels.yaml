---
version: 2
interactions:
    - id: 0
      request:
        proto: HTTP/1.1
        proto_major: 1
        proto_minor: 1
        content_length: 0
        transfer_encoding: []
        trailer: {}
        host: api.github.com
        remote_addr: ""
        request_uri: ""
        body: ""
        form: {}
        headers:
            Accept:
                - application/vnd.github.squirrel-girl-preview
            User-Agent:
                - go-github/v69.2.0
            X-Github-Api-Version:
                - "2022-11-28"
        url: https://api.github.com/repos/geropl/github-mcp-go-test/issues?direction=desc&labels=bug%2Cenhancement&per_page=30&sort=created&state=all
        method: GET
      response:
        proto: HTTP/2.0
        proto_major: 2
        proto_minor: 0
        transfer_encoding: []
        trailer: {}
        content_length: -1
        uncompressed: true
        body: '[{"url":"https://api.github.com/repos/geropl/github-mcp-go-test/issues/14","repository_url":"https://api.github.com/repos/geropl/github-mcp-go-test","labels_url":"https://api.github.com/repos/geropl/github-mcp-go-test/issues/14/labels{/name}","comments_url":"https://api.github.com/repos/geropl/github-mcp-go-test/issues/14/comments","events_url":"https://api.github.com/repos/geropl/github-mcp-go-test/issues/14/events","html_url":"https://github.com/geropl/github-mcp-go-test/issues/14","id":2902872912,"node_id":"I_kwDOOEmhcs6tBlNQ","number":14,"title":"Test Issue with Labels","user":{"login":"geropl","id":32448529,"node_id":"MDQ6VXNlcjMyNDQ4NTI5","avatar_url":"https://avatars.githubusercontent.com/u/32448529?v=4","gravatar_id":"","url":"https://api.github.com/users/geropl","html_url":"https://github.com/geropl","followers_url":"https://api.github.com/users/geropl/followers","following_url":"https://api.github.com/users/geropl/following{/other_user}","gists_url":"https://api.github.com/users/geropl/gists{/gist_id}","starred_url":"https://api.github.com/users/geropl/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/geropl/subscriptions","organizations_url":"https://api.github.com/users/geropl/orgs","repos_url":"https://api.github.com/users/geropl/repos","events_url":"https://api.github.com/users/geropl/events{/privacy}","received_events_url":"https://api.github.com/users/geropl/received_events","type":"User","user_view_type":"public","site_admin":false},"labels":[{"id":8255618595,"node_id":"LA_kwDOOEmhcs8AAAAB7BK-Iw","url":"https://api.github.com/repos/geropl/github-mcp-go-test/labels/bug","name":"bug","color":"d73a4a","default":true,"description":"Something isn''t working"},{"id":8255618610,"node_id":"LA_kwDOOEmhcs8AAAAB7BK-Mg","url":"https://api.github.com/repos/geropl/github-mcp-go-test/labels/enhancement","name":"enhancement","color":"a2eeef","default":true,"description":"New feature or request"}],"state":"open","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":0,"created_at":"2025-03-07T12:23:33Z","updated_at":"2025-03-07T12:23:33Z","closed_at":null,"author_association":"OWNER","sub_issues_summary":{"total":0,"completed":0,"percent_completed":0},"active_lock_reason":null,"body":"This is a test issue with labels created by the test suite.","closed_by":null,"reactions":{"url":"https://api.github.com/repos/geropl/github-mcp-go-test/issues/14/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0,"rocket":0,"eyes":0},"timeline_url":"https://api.github.com/repos/geropl/github-mcp-go-test/issues/14/timeline","performed_via_github_app":null,"state_reason":null}]'
        headers:
            Access-Control-Allow-Origin:
                - '*'
            Access-Control-Expose-Headers:
                - ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset
            Cache-Control:
                - private, max-age=60, s-maxage=60
            Content-Security-Policy:
                - default-src 'none'
            Content-Type:
                - application/json; charset=utf-8
            Date:
                - Fri, 07 Mar 2025 12:24:11 GMT
            Etag:
                - W/"3598947e326f30845cf749b8f4e6d8efb153fb265057155411648c0ee58632f2"
            Github-Authentication-Token-Expiration:
                - 2025-04-06 13:14:03 +0200
            Referrer-Policy:
                - origin-when-cross-origin, strict-origin-when-cross-origin
            Server:
                - github.com
            Strict-Transport-Security:
                - max-age=31536000; includeSubdomains; preload
            Vary:
                - Accept, Authorization, Cookie, X-GitHub-OTP,Accept-Encoding, Accept, X-Requested-With
            X-Accepted-Github-Permissions:
                - issues=read
            X-Content-Type-Options:
                - nosniff
            X-Frame-Options:
                - deny
            X-Github-Api-Version-Selected:
                - "2022-11-28"
            X-Github-Media-Type:
                - github.v3; param=squirrel-girl-preview
            X-Github-Request-Id:
                - FE43:3947F4:3A2188F:3BFA605:67CAE56B
            X-Ratelimit-Limit:
                - "5000"
            X-Ratelimit-Remaining:
                - "4983"
            X-Ratelimit-Reset:
                - "1741353443"
            X-Ratelimit-Resource:
                - core
            X-Ratelimit-Used:
                - "17"
            X-Xss-Protection:
                - "0"
        status: 200 OK
        code: 200
        duration: 220.116061ms
