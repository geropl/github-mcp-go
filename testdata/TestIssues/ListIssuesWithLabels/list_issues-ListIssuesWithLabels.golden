{
  "output": "# Issues\n\nFound 1 issues.\n\n## 1. Test Issue with Labels\n\n**Number:** #14  \n**State:** open  \n**Created:** Fri, 07 Mar 2025 12:23:33 UTC  \n**URL:** https://github.com/geropl/github-mcp-go-test/issues/14  \n\nThis is a test issue with labels created by the test suite.\n\n**Labels:** bug, enhancement  \n**Comments:** 0  \n\n",
  "err": ""
}